_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates seamlessly, without requiring
+  user intervention and approval.
+- As a user, I don't want to wait for the new version to be installed; I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When users visit your web app, they are not prompted with a request to update,
+they simply see yout brand new UI with all your awesome functionality. As web apps, progressive web apps
+behave the same way out of the box but provide you with the means to fully customize the
+life cycle of your app.</p>
```

Comment:
I'm trying to explain what is fresh in the context of a progressive web application. In the [original blog post](https://infrequently.org/2015/06/progressive-apps-escaping-tabs-without-losing-our-soul/), _fresh_ refer to the fact that a web site is always _up to date_. The user is never prompted about installing or updating because that is not the model of the Web. Progressive web applications, as web applications behave this way out of the box: you update the server, the user gets the new version. I think trying to communicate this is enough.

> Also, in the first sentence, "see your brand new UI" suggests that updates are UI-specific, which isn't necessarily the case. And "all your awesome functionality" sounds like pandering. I would instead say, "they simply get the updated version of the app" or "they simply get the latest version of the app" or (depending on whether or not you reference a specific update in the first half of the sentence).

Thanks for the feedback, I was trying the tone of the text. It does not work. :)